Update README.md file

JelilFaisalAbudu opened the pull request:
# Requesting code review for the phase of the ror-Scaffold project

* I have finished deploying the project to the Heroku cloud.
* I've also updated the README.md file to include the live demo as required in the project specifications.
* [Live Demo](https://jelil-ror-social-scaffold.herokuapp.com/)

## Thanks...

JelilFaisalAbudu committed:
Update README.md file

JelilFaisalAbudu committed:
Update the readme

andresporras3423 reviewed (changes requested):
### Status: Changes Required :recycle:
**Hello Faisal Abudu** 
Great job with your first submission :muscle:  :muscle: 
Your Live Demo is working, the project meets most of the requirements :clap: :clap: :clap:  
We still need to fix some minor details:
- [ ] In your _friends.html.erb file, you have a typo on line 4, replace "Fiends" with "Friends"
- [ ] In your _requested_friends.html.erb file, you have a double "%>" tag at the end of line 11, remove one
- [ ] In your README file, add a screenshot of your Live Demo

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

JelilFaisalAbudu committed:
Add the web app's screenshot

JelilFaisalAbudu committed:
Embed screenshot

JelilFaisalAbudu committed:
Correct typos

JelilFaisalAbudu commented:
# Requesting code review for the required changes made

* Added screenshot to the readme
* Corrected the typo on line 4 of the _friends.html.erb file
* Removed the extra ```%>``` on line 11 of the  _requested_friends.html.erb file

## Done...

addod19 reviewed (approved):
## Status: Approved :trophy: :traffic_light: 

Hi @JelilFaisalAbudu, congratulations on fixing all the comments outlined by the previous TSE :+1: 
The project is now approved :smile:  However, let us take note of the optional comment
[OPTIONAL]
- The deployment should be on the development branch ready to be merged into the master. Going forward, let us take note of that. :+1: 

Good luck with your next projects :1st_place_medal: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)
